File: python/7 kyu/396.py
# https://www.codewars.com/kata/5681cf0be812b41721000034/train/python

# Passed

def remove_noise(st):
    noise = set("%$&/#·@|º\ª")
    result = "".join([char for char in st if char not in noise])
    return result

output = remove_noise("h%e&·%$·llo w&%or&$l·$%d")
print(output)

File: nodejs/7 kyu/396.js
// https://www.codewars.com/kata/55d1d6d5955ec6365400006d/train/javascript

// Passed

function roundToNext5(n) {
  let remainder = n % 5;
  let result = remainder === 0 ? n : 5 * Math.ceil(n / 5);
  return result;
}

const output = roundToNext5(3);
console.log(output);